### DISCUSSION TITLE
How to select text up to a search result?

### DISCUSSION THREAD

**Discussion by gruhn** [helix-editor/helix#5846] [Q&A] [answered]
In Vim you can press `V` for block selection, search a keyword with `/`, and then keep pressing `n` to select everything up to the search result.

https://user-images.githubusercontent.com/26570572/216984983-b5976940-9ada-48f5-80b9-95e47c48c2f5.mov

In Helix, if I press `v`, and then search with `/`, I generate more and more cursors which each select a search result. 

https://user-images.githubusercontent.com/26570572/216982890-5b2d9d00-1cd6-43ef-8060-495fec19e922.mov

That's also very useful but is there a way to get something analogous to the Vim behavior? I miss this in situations where I'd like to select a large chunk of the file and I roughly know some keywords that appears at the end of my intended selection. For example when resolving a merge conflict and I want to delete everything from 
```
<<<<<<< HEAD
```

up to
```
=======
```

**Comment by the-mikedavis** [comment, marked as answer]
This would be possible with https://github.com/helix-editor/helix/issues/5672
